src/transforms/transform-deployment.ts

```diff
@@ -101,7 +100,7 @@ async function getCommitMessagesSinceLastSuccessfulDeployment(
 // https://developer.atlassian.com/cloud/jira/software/rest/api-group-builds/#api-deployments-0-1-bulk-post
 // Deployment state - Jira: Can be one of unknown, pending, in_progress, cancelled, failed, rolled_back, successful
 function mapState(state: string): string {
-	switch (state) {
+	switch (state?.toLowerCase()) {
```

Comment:
if the state an be undefined, then you should specify it in the params with an optional annotation.